python/lsst/pipe/tasks/functors.py

```diff
@@ -735,50 +746,56 @@ def computePixel(row):
 
 
 def fluxName(col):
+    """Append _instFlux to the column name if it doesn't have it already."""
     if not col.endswith('_instFlux'):
         col += '_instFlux'
     return col
 
 
 def fluxErrName(col):
+    """Append _instFluxErr to the column name if it doesn't have it already."""
     if not col.endswith('_instFluxErr'):
         col += '_instFluxErr'
     return col
 
 
 class Mag(Functor):
-    """Compute calibrated magnitude
+    """Compute calibrated magnitude.
 
-    Takes a `calib` argument, which returns the flux at mag=0
-    as `calib.getFluxMag0()`.  If not provided, then the default
-    `fluxMag0` is 63095734448.0194, which is default for HSC.
-    This default should be removed in DM-21955
+    Returns the flux at mag=0.
+    The default ``fluxMag0`` is 63095734448.0194, which is default for HSC.
+    TO DO: This default should be made configurable in DM-21955.
 
     This calculation hides warnings about invalid values and dividing by zero.
 
-    As for all functors, a `dataset` and `filt` kwarg should be provided upon
-    initialization.  Unlike the default `Functor`, however, the default dataset
-    for a `Mag` is `'meas'`, rather than `'ref'`.
+    As with all functors, a ``dataset`` and ``filt`` kwarg should be provided
+    upon initialization.
+    Unlike the default `Functor`, however, the default dataset for a `Mag` is
+    ``'meas'``, rather than ``'ref'``.
 
     Parameters
     ----------
     col : `str`
-        Name of flux column from which to compute magnitude.  Can be parseable
-        by `lsst.pipe.tasks.functors.fluxName` function---that is, you can pass
-        `'modelfit_CModel'` instead of `'modelfit_CModel_instFlux'`) and it will
-        understand.
-    calib : `lsst.afw.image.calib.Calib` (optional)
-        Object that knows zero point.
+        Name of flux column from which to compute magnitude.
+        Can be parseable by `~lsst.pipe.tasks.functors.fluxName` function; that
+        is, you can pass ``'modelfit_CModel'`` instead of
+        ``'modelfit_CModel_instFlux'``, and it will understand.
     """
     _defaultDataset = 'meas'
 
     def __init__(self, col, calib=None, **kwargs):
         self.col = fluxName(col)
         self.calib = calib
         if calib is not None:
+            # TO DO: DM-39914 Remove deprecated calib argument in Mag functor.
             self.fluxMag0 = calib.getFluxMag0()[0]
+            warnings.warn(
+                "Config field 'calib' is deprecated, and will be removed after v27.",
```

Comment:
I don't think `Config field` is applicable here. It should perhaps be "`calib` argument".

Reply:
Changed  to `The 'calib' argument is deprecated, and will be removed after v27.`.